### PR TITLE
correct pm2.5 formula to show value on display

### PR DESCRIPTION
**Pull request by pnt325**
Update the PM2.5 calculate formula #182 . The correct value show on device display or use on local only.

### CODE REVIEW

File: src/AgOledDisplay.cpp [hunk 1]
```diff
@@ -304,6 +304,10 @@ void OledDisplay::showDashboard(const char *status) {
       DISP()->drawStr(55, 27, "PM2.5");
 
       /** Draw PM2.5 value */
+      int pm25 = value.pm25_1;
+      if (config.hasSensorSHT) {
+        pm25 = ag->pms5003.pm25Compensated(pm25, value.Humidity);
```
Comment (nick-4711) [inline src/AgOledDisplay.cpp:309]: propose to rename the method to simply .compensate(). We can see from the params that this is related to pm25.

File: src/PMS/PMS.cpp [hunk 2]
```diff
@@ -270,6 +270,41 @@ int PMSBase::pm25ToAQI(int pm02) {
     return 500;
 }
 
+/**
+ * @brief Correction PM2.5
+ * 
+ * @param pm25 Raw PM2.5 value
+ * @param humidity Humidity value (%)
+ * @return float
```
Comment (nick-4711) [inline src/PMS/PMS.cpp:278]: it returns an int? Propose to write

@return the compensated PM2.5 value

File: src/PMS/PMS.cpp [hunk 3]
```diff
@@ -270,6 +270,41 @@ int PMSBase::pm25ToAQI(int pm02) {
...
 
+/**
+ * @brief Correction PM2.5
+ * 
+ * @param pm25 Raw PM2.5 value
+ * @param humidity Humidity value (%)
+ * @return float 
+ */
+int PMSBase::pm25Compensated(int pm25, float humidity) {
+  float value;
```
Comment (nick-4711) [inline src/PMS/PMS.cpp:281]: We need to link to the documentation where this is from

File: src/AgOledDisplay.cpp (hunk 1 above)
Comment (pnt325) [reply src/AgOledDisplay.cpp:309]: Ok, I'll change the name and update API comment to mention the input is PM2.5

File: src/PMS/PMS.cpp (hunk 2 above)
Comment (pnt325) [reply src/PMS/PMS.cpp:278]: Yes, it's return the int. I'll update the method comment

File: src/PMS/PMS.cpp (hunk 3 above)
Comment (pnt325) [reply src/PMS/PMS.cpp:281]: @airgradienthq  please add your document for this formula.

**Pull request merged**